.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,54 @@
+name: E2E test
+on:
+  pull_request:
+      branches:
+        - master
+  workflow_dispatch:
+    environment:
+      description: 'Environment to run tests against'
+      type: environment
+      required: true 
+env:
+  TEST_USERNAME: admin
+  TEST_PASSWORD: password
+  CATTLE_BOOTSTRAP_PASSWORD: password
+  TEST_BASE_URL: https://127.0.0.1:8005
+  API: https://127.0.0.1
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        if: "!contains( github.event.pull_request.labels.*.name, 'ci/skip-e2e')"
```

Comment:
The whole pipeline will fail if there's not a job.

I was not able to identify a way to avoid repetition for the condition by holding it in a variable.

Reply:
Had a look, couldn't find any way to ignore on label on the top level workflow. If we need a job then we can always to the flip with a fake job+step. Something like

```
jobs:
  test:
    if: "!contains( github.event.pull_request.labels.*.name, 'ci/skip-e2e')"
    runs-on: ubuntu-latest
    steps:
    ...
  skip:
    if: "contains( github.event.pull_request.labels.*.name, 'ci/skip-e2e')"
    runs-on: ubuntu-latest
    steps:
   - name: Skipping e2e tests
        run: echo "Skipping..."
```

Reply:
As already experienced, we'll have always a skipped "skip-e2e" check for each PR.

![Screenshot 2022-05-19 at 14 16 47](https://user-images.githubusercontent.com/5009481/169291217-6b2f1bce-d9ae-4e64-a40d-d6985ca2d777.png)